homeassistant/components/samsungtv/bridge.py

```diff
@@ -191,6 +192,7 @@ def __init__(self, method, host, port, token=None):
         """Initialize Bridge."""
         super().__init__(method, host, port)
         self.token = token
+        self.default_port = 8001
```

Comment:
What if the default port doesn't work?

Reply:
Sorry I don't understand your question. As I said: 0.106 do currently support only websocket TVs running on port 8001. All current config entries have port = None. Port None = port 8001.
If a TV requires port 8002 it's currently not supported so there are no config entries.
Default port applies only to currently supported websocket Samsung TVs. That are all running on port 8001.

Reply:
Ok. Thanks for the clarification.